tests/test_example_dags.py

```diff
@@ -71,6 +72,11 @@ def get_dag_bag() -> DagBag:
             print(f"Adding {dagfile} to .airflowignore")
             file.writelines([f"{dagfile}\n"])
 
+        # Python 3.8 has reached its end of life (EOL), and dbt no longer supports this version.
```

Comment:
Are there any specific versions of dbt that no longer support Python 3.8? Is it just the latest or did they backport the removal of Python 3.8 support?

Reply:
It appears this was removed in version 1.7.19, as noted in the changelog: https://github.com/dbt-labs/dbt-core/blob/1.7.latest/CHANGELOG.md#under-the-hood. Older versions of dbt-core still support Python 3.8 according to PyPI. However, the compatibility matrix on their documentation states support for Python versions greater than 3.8: https://docs.getdbt.com/faqs/Core/install-python-compatibility.